src/core/server/saved_objects/migrations/core/document_migrator.ts

```diff
@@ -51,12 +51,15 @@ import type {
   SavedObjectsMigrationVersion,
   SavedObjectsNamespaceType,
 } from '@kbn/core-saved-objects-common';
-import { SavedObjectUnsanitizedDoc } from '../../serialization';
-import { SavedObjectsType } from '../../types';
+import type {
+  SavedObjectUnsanitizedDoc,
+  SavedObjectsType,
+  ISavedObjectTypeRegistry,
```

Comment:
Comment: We're not very consistent with our use of the `I` prefix to indicate internal types.